### ISSUE TITLE
__init__.py is empty from the "utils" folder.

### ISSUE THREAD

**Issue by ShenZC25** [Dekai21/Multi_Agent_Intersection#6]
Hello, thank you for your excellent work. When I was running the program, I found the following error in `sumo_simulation.py`

![image](https://github.com/Dekai21/Multi_Agent_Intersection/assets/89288084/9a693e01-485e-42f2-b011-7c71638e60b3)

Later, it was found in the `utils` folder that the` __init__.py` file was empty. What is the reason? How to solve it?

thank you

**Comment by Dekai21** [comment]
Hi, I didn't meet this problem before. It looks like the relative path problem problem. Since the utils.py file is already provided in this repository, I would recommend you to check the relative path.

**Comment by ShenZC25** [comment]
> 嗨，我以前没有遇到过这个问题。它看起来像相对路径问题。由于此存储库中已经提供了 utils.py 文件，因此建议您检查相对路径。

Hello, thank you very much for your answer. Now I encounter some new problems, that is, I can't reproduce the effect of the given model. Could you please tell me how to set some specific parameters so that I can reproduce them?
For example, batch size, epoch, num_mpc_aug of train set and val set et al.

Here are some of my parameters:
python generate_csv.py --num_seconds 1000 --split train    
python generate_csv.py --num_seconds 1000 --split test  
python preprocess.py --csv_folder csv/train --pkl_folder csv/train_pre --num_mpc_aug 2
python preprocess.py --csv_folder csv/val --pkl_folder csv/val_pre --num_mpc_aug 0 
python train_gnn.py --train_folder csv/train_pre --val_folder csv/val_pre --epoch 2000 --exp_id sumo_1225 --batch_size 800  

And here are some of the results:

![image](https://github.com/Dekai21/Multi_Agent_Intersection/assets/89288084/a5d3421b-9ab3-4ea7-9543-a2c1fc7a020f)
![image](https://github.com/Dekai21/Multi_Agent_Intersection/assets/89288084/bc4e7ff5-790c-4137-9ac7-fb059bf80f1a)


Thank you.

**Comment by Angelnian** [comment]
> > 嗨，我以前没有遇到过这个问题。它看起来像相对路径问题。由于此存储库中已经提供了 utils.py 文件，因此建议您检查相对路径。
> 
> Hello, thank you very much for your answer. Now I encounter some new problems, that is, I can't reproduce the effect of the given model. Could you please tell me how to set some specific parameters so that I can reproduce them? For example, batch size, epoch, num_mpc_aug of train set and val set et al.
> 
> Here are some of my parameters: python generate_csv.py --num_seconds 1000 --split train python generate_csv.py --num_seconds 1000 --split test python preprocess.py --csv_folder csv/train --pkl_folder csv/train_pre --num_mpc_aug 2 python preprocess.py --csv_folder csv/val --pkl_folder csv/val_pre --num_mpc_aug 0 python train_gnn.py --train_folder csv/train_pre --val_folder csv/val_pre --epoch 2000 --exp_id sumo_1225 --batch_size 800
> 
> And here are some of the results:
> 
> ![image](https://private-user-images.githubusercontent.com/89288084/292757874-a5d3421b-9ab3-4ea7-9543-a2c1fc7a020f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU0NDE5MzEsIm5iZiI6MTcyNTQ0MTYzMSwicGF0aCI6Ii84OTI4ODA4NC8yOTI3NTc4NzQtYTVkMzQyMWItOWFiMy00ZWE3LTk1NDMtYTJjMWZjN2EwMjBmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA0VDA5MjAzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcyMWUxY2Y0NzI4ZmU2NThhZWJiM2IxYWM2ZWExMGQ2NzEzMDdjMmQwOTA0OGJiNmFhN2E5OTY3YzNlYTE5NjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.veZePgYaEsg_RuibzO9QfZ-8I_hE77Mlv3shxKPCYIM) ![image](https://private-user-images.githubusercontent.com/89288084/292757910-bc4e7ff5-790c-4137-9ac7-fb059bf80f1a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU0NDE5MzEsIm5iZiI6MTcyNTQ0MTYzMSwicGF0aCI6Ii84OTI4ODA4NC8yOTI3NTc5MTAtYmM0ZTdmZjUtNzkwYy00MTM3LTlhYzctZmIwNTliZjgwZjFhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA0VDA5MjAzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFjODMzM2MxNmFmZWFlNDJiOTJhNmFlZDBmMGYxZGM4ZmViMDFhNzBhYzM5NGMyNWJlYjMzMjNmYWU5MDM0MGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.emnAzfoNI_Blm_9WB4M1fV49v6nstkkDs2RXHl3aIhA)
> 
> Thank you.

hi,during the training, I also encountered the problem of excessive training loss and verification loss. Have you solved the problem of parameters?

**Comment by Dekai21** [comment]
This issue seems to be related to a relative path problem. The vehicle crash issue has also been alleviated in [this commit](https://github.com/Dekai21/Multi_Agent_Intersection/commit/2b524a93183ca7c2c0d246a4bbc1f9672511860b).
